### PR TITLE
Egym iwatch

### PR DESCRIPTION
**Pull request by retif**
User is able to just enable module and get completely finished landing page, equal to wireframes.

Steps for review are next.
1. Go to extensions and enable this module.
2. Once module would be enabled You will see status message with a link to just generated landing page.
3. Click to this link and check that landing page exist and equal to wireframes.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

**Comment by podarok** [comment]
@paige-kiecker Looks like I missed something
Should we have this in OpenY?

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1943/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1943/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1943/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1943/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1943/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1943/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1943/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1943/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1943/

**Comment by podarok** [comment]
@retif and @anpolimus 
IS there any way for you to improve steps for review
It is not obvious how to test this code
We used to have a best practice for step by step review.
Appreciate attention to details from your end

File: modules/openy_features/openy_egym_iwatch/dist/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  \**************************/
+/*! no exports provided */
+/*! all exports used */
+/*! ModuleConcatenation bailout: Module is an entry point */function(e,t,r){"use strict";r.r(t);r(/*! ./src/scss/openy_egym_iwatch.scss */1)},
+/*!*****************************************!*\
+  !*** ./src/scss/openy_egym_iwatch.scss ***!
+  \*****************************************/
+/*! no static exports found */
+/*! ModuleConcatenation bailout: Module is not an ECMAScript module */function(e,t,r){}]);
+//# sourceMappingURL=css/openy_egym_iwatch.css.map
```
Comment (podarok) [inline modules/openy_features/openy_egym_iwatch/dist/main.js]: could you compile without sourcemap, please?

**Comment by anpolimus** [comment]
Hi, @podarok .
Steps for review are next.
1. Go to extensions and enable this module.
2. Once module would be enabled You will see status message with a link to just generated landing page.
3. Click to this link and check that landing page exist and equal to wireframes.

File: modules/openy_features/openy_egym_iwatch/openy_egym_iwatch.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+openy_egym_iwatch:
+  version: 1.x
+
+  css:
+    theme:
+      dist/css/openy_egym_iwatch.css: {}
```
Comment (podarok) [inline modules/openy_features/openy_egym_iwatch/openy_egym_iwatch.libraries.yml]: add `minified: true` please

File: modules/openy_features/openy_egym_iwatch/webpack.common.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  entry: './webpack-entry.js',
+  plugins: [
+    new SassLintPlugin({
+      files: path.resolve(__dirname) + '/src/**/*.scss'
+    }),
+    new MiniCssExtractPlugin({
+      filename: "./css/openy_egym_iwatch.css",
+      sourceMap: true,
+      options: {
+        sourceMap: true,
```
Comment (podarok) [inline modules/openy_features/openy_egym_iwatch/webpack.common.js]: let's have `false` for prod

**Review by podarok — CHANGES_REQUESTED** [review]
Small fixes to get this merged, please

**Comment by anpolimus** [comment]
@podarok here is little demo: https://www.loom.com/share/b79b602af8bc482aa01daef5e2cf1921

File: modules/openy_features/openy_egym_iwatch/webpack.common.js (hunk 3 above)
Comment (anpolimus) [reply modules/openy_features/openy_egym_iwatch/webpack.common.js]: done

File: modules/openy_features/openy_egym_iwatch/openy_egym_iwatch.libraries.yml (hunk 2 above)
Comment (anpolimus) [reply modules/openy_features/openy_egym_iwatch/openy_egym_iwatch.libraries.yml]: done

File: modules/openy_features/openy_egym_iwatch/dist/main.js (hunk 1 above)
Comment (anpolimus) [reply modules/openy_features/openy_egym_iwatch/dist/main.js]: Let us recompile css in prod mode

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1946/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1946/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1946/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1946/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1946/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1946/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1946/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1946/

**Comment by anpolimus** [comment]
@podarok we made all requested fixes

**Comment by gundevel** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1946/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1947/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1947/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1947/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1947/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1947/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1947/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1947/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1947/

**Comment by gundevel** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1947/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1966/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1966/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1966/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1966/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1966/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1966/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1966/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1966/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1969/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1969/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1969/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1969/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1969/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1969/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1969/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1969/

**Comment by gundevel** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1969/

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1990/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1990/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1990/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1990/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1990/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1990/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1990/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1990/

**Comment by gundevel** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1990/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2052/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2052/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2053/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2053/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2054/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2054/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2053/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2053/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2053/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2053/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2053/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2054/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2054/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2054/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2054/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2054/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2052/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2053/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2054/

**Comment by podarok** [comment]
image
image

**Pull request merged**